src/refmt_impl.ml

```diff
@@ -167,12 +191,14 @@ let () =
         )
         | Some "binary"
         | None -> fun (ast, comments) -> (
-          output_string stdout Config.ast_intf_magic_number;
-          output_value  stdout filename;
-          output_value  stdout ast
+            let (module V) = !ocaml_version in
```

Comment:
You'll have to figure out a way to pass in `ocaml_version` when this gets brought up to speed with master.